Refresh.GameServer/Database/GameDatabaseContext.Levels.cs

```diff
@@ -27,6 +28,32 @@ public void AddLevel(GameLevel level)
         });
     }
 
+    public GameLevel GetStoryLevelById(int id, TokenGame game)
```

Comment:
Developer ids aren't unique per game, we probably don't need to pass in `TokenGame` for retrieval

Reply:
PSP IDs start from 0, so i assume at least with LBP PSP, there would be clash with LBP1, assuming those start with `0` too (not sure)

Reply:
They don't, LBP1 should start somewhere in the thousands range